### PR TITLE
MODULES: Add maxbuckets to the kibana config json

### LINKED ISSUE #7450: MODULES: "Metrics: Max buckets exceeded" - increase the number of buckets
For the metrics visualisation when computing the events per second; 

Changed it to 20000; but if we’re thinking more than 30 days, the buckets will need to be adjust to 30 days x 24 hours x 60 minutes x 60 seconds = 2592000.

### PR DESCRIPTION
**Pull request by guyboertje**
Fixes #7450

### CODE REVIEW

File: logstash-core/lib/logstash/modules/kibana_config.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ def initialize(modul, settings)
     @name = modul.module_name
     @settings = settings
     @index_name = settings.fetch("dashboards.kibana_index", ".kibana")
+    @metrics_max_buckets = settings.fetch("dashboards.metrics_max_buckets", 60000)
```
Comment (suyograo) [inline logstash-core/lib/logstash/modules/kibana_config.rb]: Let's make it 50,000

Comment (guyboertje) [inline logstash-core/lib/logstash/modules/kibana_config.rb]: We should use the formula that Nic proposed as its events per second - 24 hours is 86400 seconds = 86400 buckets

Comment (suyograo) [inline logstash-core/lib/logstash/modules/kibana_config.rb]: sounds good. can you please add a comment explaining the magic number? Also, please make it a constant.

Comment (guyboertje) [inline logstash-core/lib/logstash/modules/kibana_config.rb]: Although I have made it so the user can alter the value in the modules section of LS settings, we need to clarify with SA team whether this is a module specific KV pair.

**Pull request merged**